share/pot/import.sh

```diff
@@ -23,8 +23,8 @@ _fetch_pot()
 	local _filename
 	_filename="${1}_${2}.xz"
 	if ! _fetch_pot_internal "$1" "$2" "$3" ; then
-		# remove the artifact and retry only once
-		rm -f "${POT_CACHE}/$_filename"
+		# remove the artifacts and retry, but only once
+		rm -f "${POT_CACHE}/$_filename" "${POT_CACHE}/${_filename}.meta" "${POT_CACHE}/${_filename}.skein"
```

Comment:
Even though technically not necessary, putting the first $_filename in curly braces would make this easier to parse for the human eye (that way it's easier to notice that the prefix is identical). Maybe also consider newlines, to make it even more clear.

```sh
rm -f \
  "${POT_CACHE}/${_filename}" \
  "${POT_CACHE}/${_filename}.meta" \
  "${POT_CACHE}/${_filename}.skein"
```

I assume that it's a bit late (also in terms of backwards compatibility) - having a file extension (something like .potimg) would be nice.